main.py

```diff
@@ -698,6 +698,8 @@ def ui_access_wrapper(self, *args, **kwargs):
         collection_editor.ExplorationMetadataSearchHandler),
     get_redirect_route(
         r'/explorationdataextractionhandler', admin.DataExtractionQueryHandler),
+    get_redirect_route(
+        r'/SendDummyMailToAdminHandler', admin.SendDummyMailToAdminHandler),
```

Comment:
The casing of the url does not follow the surrounding ones. **Please read the surrounding code** and follow the existing code style patterns.

Reply:
Done